releaseNotes/AzureAppConfigurationProviderPython.md

```diff
@@ -4,9 +4,9 @@
 
 ## 2.0.0b2 - October 14, 2024
 
-### Features
+### Enhancement
```

Comment:
We use both Features and Enhancement in this file. Might be good to use `Enhancements` (plural) for everything going forward. We do the same for other language release notes.